docs/requirements.txt

```diff
@@ -1,6 +1,6 @@
 sphinx_rtd_theme
 numpy
 scipy
-matplotlib
+matplotlib >=3.3.0,<=3.4.2
```

Comment:
Maybe we should add a comment somewhere about this requirement as it may be a temporary issue of matplotlib version 3.4.3?

Reply:
OK, will do. I imagine it is a temporary issue.